### DISCUSSION TITLE
Color-customizable icons?

### DISCUSSION THREAD

**Discussion by Br3nnabee** [anyproto/contributors#169] [Anytype apps]
I've recently switched to anytype and I've been LOVING it. The only thing that I've really been missing is custom icons, not as in importing images but having inbuilt support for some svg based icons like material icons etc. to be able to colour code icons, or maybe even change them in other ways.

**Comment by Bapicean** [comment]
I agree, Im missing it as well, they have it in notion.

Also im missing many features like holiday planner, everything comes up quicker in notion than Anytype
